### PR TITLE
update setVestingStakes

### PR DESCRIPTION
**Pull request by cwsnt**

### CODE REVIEW

File: contracts/governance/Staking/modules/StakingVestingModule.sol [hunk 1]
```diff
@@ -49,16 +49,38 @@ contract StakingVestingModule is IFunctionsList, StakingShared {
...
     function _setVestingStake(uint256 lockedTS, uint96 value) internal {
-        //delete all checkpoints (shouldn't be any during the first initialization)
+        require(
+            lockedTS > kickoffTS,
+            "Invalid lock dates: must greater than contract creation timestamp"
+        );
+
+        // locked date must be multiples of 14 days
+        require(
+            (lockedTS - kickoffTS) % 14 days == 0,
```
Comment (ororopickpocket) [inline contracts/governance/Staking/modules/StakingVestingModule.sol:59]: please use the TWO_WEEKS constant which used everywhere else in the contract

**Review by ororopickpocket — CHANGES_REQUESTED** [review]
Looks good, just one small change request and 2 test cases are missing (see jira ticket).

File: contracts/governance/Staking/modules/StakingVestingModule.sol (hunk 1 above)
Comment (cwsnt) [reply contracts/governance/Staking/modules/StakingVestingModule.sol]: Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/472/commits/871541179e096327e612ede25477204ffae4dc87)

**Review by jameshowlett977 — APPROVED** [review]

**Comment by jameshowlett977** [comment]
LGTM

**Review by ororopickpocket — APPROVED** [review]

**Pull request merged**